testsuite/tests/typing-unique/unique.ml

```diff
@@ -594,16 +594,16 @@ Line 2, characters 11-19:
 
 let array_pats (arr : int option array) =
   match arr with
-  | [| o |] -> let _ = unique_id arr in aliased_id o
+  | [| o |] -> let _ = unique_id arr in None
```

Comment:
I need to change this bit, since `o` no longer has `many` modality, and thus will still be tracked by UA, which would cause type error that pre-empt the expected error. That means our UA is made a bit too strict.

@anfelor please confirm this is acceptable.

Reply:
I think the difference you are refering to no longer exists: `mutable` no longer implies `many`, which makes UA still tracks the usage of `o`, and causing this test to change.

I will revert this change - `mutable` will still imply `many`.

In the long run we need to finish https://github.com/oxcaml/oxcaml/pull/2377 so that modalities/mode crossing are tracked in a finer way.